cfe_internal/enterprise/federation/federation.cf

```diff
@@ -366,11 +352,19 @@ body contain cfpostgres_user
 bundle agent exported_data
 # @brief Run script to dump pg data on feeder hub
 {
+  methods:
```

Comment:
Since methods come before commands in the normal order https://docs.cfengine.com/docs/3.12/reference-language-concepts-normal-ordering.html this should result in inventory being refreshed before dump is run.  And i see that additionally, you depended the dump on mpf_fresh_inventory_view, which is the actual command that gets run inside the cfe_internal_refresh_inventory_view bundle. 

I might add a handle to the Refresh Inventory promise and then change the depends on to depend on the whole bundle. But this looks fine to me